remix-tests/src/compiler.js

```diff
@@ -52,7 +51,7 @@ function compileFileOrFiles (filename, isDirectory, cb) {
   ], function (err, result) {
     let errors = (result.errors || []).filter((e) => e.type === 'Error' || e.severity === 'error')
     if (errors.length > 0) {
-      signale.fatal(errors)
+      cb(errors)
```

Comment:
On one of my next path I wanted to only keep `return cb(errors)`. This error later can be shown to users to describe why compilation failed.